google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/InsertAllRequest.java

```diff
@@ -202,8 +215,11 @@ public Builder addRow(String id, Map<String, ?> content) {
     }
 
     /**
-     * Adds a row to be inserted without an associated id. Please notice that data for fields of
-     * type {@link LegacySQLTypeName#BYTES} must be provided as a base64 encoded string.
+     * Adds a row to be inserted without an associated id.
+     *
+     * <p>To ensure proper serialization of numeric data, it is recommended to supply values using a
```

Comment:
per google tech writing style, avoid passive voice; that is, "it is recommended to supply values" --> "supply values"